fdbrpc/RESTUtils.actor.cpp

```diff
@@ -237,37 +237,31 @@ TEST_CASE("fdbrpc/RESTUtils") {
 	}
 
 	// valid URI with service
-	try {
+	{
```

Comment:
One thing I just realized now: the test name should start with a `/`.

I think @sfc-gh-satherton asked this before: Each of these blocks is a test. Why not make them tests? You could then create a new test file in `tests/fast` that runs all your rest utils tests. Some like this:

1. Create one test for each block and prefix it with `/fdbrpc/RESTUtils` (for example `/fdbrpc/RESTUtils/ValidURIWithService`.
2. create `/tests/fast/RESTUtilsUnit.toml` with the following contents:

```
testTitle=UnitTests
startDelay=0
useDB=false

    testName=UnitTests
    testsMatching=`/fdbrpc/RESTUtils`
```

Reply:
Apologies I might have skipped Steve's advice. Thanks for pointing out.

I will apply the same principle to RESTKmsConnector test cases as well, I kept them all along to avoid having to type all individual test cases name. With pattern matching, it makes sense to split them into individual tests.